File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-03/plagiarized/L1/07/Kasus3L1.kt
package IR

import java.util.*

/*
* To change this license header, choose License Headers in Project Properties.
* To change this template file, choose Tools | Templates
* and open the template in the editor.
*/

/**
 *
 * @author B15130F5DDB6B5F1622EF91DAC4C1AAE
 */
object Kasus3L1 {
    @JvmStatic
    fun main(args: Array<String>) {
        val input = Scanner(System.`in`)

        print("Enter weight in pounds: ")
        val weight = input.nextDouble()

        print("Enter feet: ")
        val feet = input.nextDouble()
        print("Enter inches: ")
        val inches = input.nextDouble()
        val height = feet * 12 + inches

        val bmi = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

        println("BMI is $bmi")
        if (bmi < 18.5) println("Underweight")
        else if (bmi < 25) println("Normal")
        else if (bmi < 30) println("Overweight")
        else println("Obese")
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L1/07/Kasus3L1.py
import math

# Get user input
weight_lbs = float(input("Enter weight in pounds: "))
feet = float(input("Enter feet: "))
inches = float(input("Enter inches: "))

# Calculate height in meters
height_m = (feet * 12 + inches) * 0.0254

# Calculate BMI
weight_kg = weight_lbs * 0.45359237
bmi = weight_kg / (height_m ** 2)

# Print BMI and classification
print(f"BMI is {bmi:.2f}")
if bmi < 18.5:
    print("Underweight")
elif bmi < 25:
    print("Normal")
elif bmi < 30:
    print("Overweight")
else:
    print("Obese")

